### PR TITLE
Add providers

### LINKED ISSUE #36: Add tile providers
Should we also add some tile providers? It would be good to have an unopiniated list we can refer people to.

### PR DESCRIPTION
**Pull request by dschep**
closes #36

I did just names alphabetically without descriptions to minimize bias.

I'm sure more exist that I don't know of.

### CODE REVIEW

**Review by HarelM — APPROVED** [review]

**Comment by HarelM** [comment]
@wipfli please take a look, as your last comment in the linked issue seems like there's still some stuff that needs to be sorted out...

File: README.md [hunk 1]
```diff
@@ -75,6 +75,17 @@ A collection of awesome things that use or support [MapLibre](https://maplibre.o
...
+
+- [Esri](https://developers.arcgis.com/maplibre-gl-js/)
+- [Geofabrik](https://www.geofabrik.de/maps/vectortiles.html)
+- [Geoapify](https://www.geoapify.com/map-tiles/)
+- [JawgMaps](https://www.jawg.io/)
+- [MapTiler](https://www.maptiler.com/)
+- [OSM Americana Community Vector Tile Server](https://tile.ourmap.us/)
+- [Protomaps](https://protomaps.com/)
+- [Stadia Maps](https://stadiamaps.com/)
+
```
Comment (wipfli) [inline README.md:88]:
```suggestion
## Map/Tile Providers

- [**Amazon Location Services**](https://aws.amazon.com/location/)
- [**Azure Maps**](https://azure.microsoft.com/en-us/products/azure-maps)
- [Esri](https://developers.arcgis.com/maplibre-gl-js/)
- [Geofabrik](https://www.geofabrik.de/maps/vectortiles.html)
- [Geoapify](https://www.geoapify.com/map-tiles/)
- [HERE](https://www.here.com/docs/bundle/vector-tile-api-developer-guide/page/README.html)
- [**JawgMaps**](https://www.jawg.io/)
- [Mapbox](https://www.mapbox.com/)
- [MapTiler](https://www.maptiler.com/)
- [Mierune](https://www.mierune.co.jp/?lang=en)
- [OSM Americana Community Vector Tile Server](https://tile.ourmap.us/)
- [Protomaps](https://protomaps.com/)
- [Stadia Maps](https://stadiamaps.com/)
- [TomTom](https://www.tomtom.com/products/maps/)

**In bold**: Members of the [MapLibre Sponsorship Program](https://maplibre.org/sponsors/)

```

**Review by wipfli — APPROVED** [review]

**Comment by wipfli** [comment]
We can have this list in the awesome-maplibre repo. I added some more tile service providers and highlighted the ones that are part of the MapLibre Sponsorship Program...

**Comment by louwers** [comment]
Good solution

**Pull request merged**